mkdocs.yml

```diff
@@ -178,6 +178,5 @@ pages:
 - Google Site Verification: ./google64817fdc11b2f6b6.html
 
 theme: readthedocs
-theme_dir: glustertheme
```

Comment:
Is specifying `theme_dir` no longer required or the contents of that directory also isn't required ? It contains the fix you submitted long ago to properly form canonical URL.

Reply:
It appears that `theme_dir` is no longer a valid entry. Updated it with the current way of doing it.